### PR TITLE
Features/hoverlabelheight

### LINKED ISSUE #23: get vertical position of label closer to cursor for upper and lower quadrant
Right now the vertical label height isn't closer to the cursor especially if there is no "parenthesis data". 
```
      setHoveredLabelY(ev.clientY - + rect.y - 100);
```

We should adjust it it's always close to the cursor using the reference height.

### PR DESCRIPTION
**Pull request by Strattonshrugged**
Added height variable to read label reference, adjusts vertical position of label to be closer to cursor on names w/o an origin

fixes #23 

Also added metadata and image files for warp-core-cafe

marioSnippet

### CODE REVIEW

File: src/BusyImage.tsx [hunk 1]
```diff
@@ -82,14 +82,15 @@ const BusyImage = (props: BusyImageProps) => {
...
     if (ev.clientX + ev.currentTarget.scrollLeft > ev.currentTarget.scrollWidth/2){
       setHoveredLabelX(ev.clientX - rect.x + ev.currentTarget.scrollLeft - maxLabelWidth);
     } else {
       setHoveredLabelX(ev.clientX - rect.x + ev.currentTarget.scrollLeft + 10);
     };
     if (ev.clientY - rect.y > ev.currentTarget.scrollHeight/2){
-      setHoveredLabelY(ev.clientY - + rect.y - 100);
+      setHoveredLabelY(ev.clientY - rect.y - labelHeight);
     } else {
       setHoveredLabelY(ev.clientY - rect.y + 10);
```
Comment (strattonbrazil) [inline src/BusyImage.tsx]: I think both upper and lower halves of the image should take the label height into account. There's less gap in the upper section, but it's inconsistent depending on if there's extra data.

**Review by strattonbrazil — CHANGES_REQUESTED** [review]
Change looks good, but I think we should make a similar change for the other y-check. 

Also, I think the warp-core-cafe.jpg image is too low of a resolution. It also waits much of that density on that large border. Can we use [this asset](http://www.jeffcarlisle.com/files/images/CharCon2018_05B_forweb01E.jpg)? It drops the border while still having the artist attribution. I think the existing size is probably fine, too.

**Comment by Strattonshrugged** [comment]
kk, I'll dump the border and make sure we keep the artists name.

Also I think I just updated a pull request the same instant you sent this.
Weird timing.

**Comment by Strattonshrugged** [comment]
I guess I'm confused about taking the labelheight into account if the cursor is in the top half, I just dropped the +10 adjustment and it seems pretty tight without overlapping the label.

The +10 x-axis adjustment still exists on the left to keep the cursor from overlapping the label. If you want this closer on the left-bottom I can make it check for which quartile it's in instead of just top/bottom left/right, it'd be pretty easy I think.

**Comment by strattonbrazil** [comment]
I must have been testing on another branch. Changes look good.

**Pull request merged**